src/components/createevents.js

```diff
@@ -39,28 +46,44 @@ class createEvent extends Component {
     this.setState({ [e.target.name]: e.target.value });
   };
 
+  streetGenerator = (str_one, str_two) => {
+      if(!str_two) {
+        return str_one
+      } else {
+        return `${str_one}, ${str_two}`
+      };
+  }
+
   handleCreation = async () => {
     try {
       if (!this.state.activity_id) {
         alert("Please select an activity!");
       }
       const {
-        activity_id,
-        event_date_start,
-        event_date_end,
-        is_public_event,
-        max_player
-      } = this.state;
-      const creator_id = this.state.creator.user_id;
-      const body = {
         activity_id,
         event_date_start,
         event_date_end,
         is_public_event,
         max_player,
-        creator_id
+        message,
+        locationStreet_one,
+        locationStreet_two,
+        locationCity,
+        locationZip,
+        locationState
+        } = this.state;
```

Comment:
@DrewStableBoi Is this supposed to be a this.setState() ?

Reply:
@Paulyg15 No, I'm destructuring all those fields from state within the context of the method so I don't have to use this.state."" every time. Make sense?

Reply:
@DrewStableBoi , Ya you are totally right! I read that backwards lol. Math is hard

Reply:
BIG MOOD, brother. Haha it's all good!